hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -60,6 +61,19 @@ public record EthTxData(
     static final BigInteger LEGACY_V_BYTE_SIGNATURE_0 = BigInteger.valueOf(27);
     static final BigInteger LEGACY_V_BYTE_SIGNATURE_1 = BigInteger.valueOf(28);
 
+    static final int FOUNDRY_DETERMINISTIC_DEPLOYER_GAS_PRICE_MULTIPLIER = 100;
+    // The specific transaction bytes that are used to deploy the Foundry Deterministic Deployer contract
+    public static final byte[] FOUNDRY_DETERMINISTIC_DEPLOYER_TRANSACTION;
+
+    static {
+        try {
+            FOUNDRY_DETERMINISTIC_DEPLOYER_TRANSACTION = Hex.decodeHex(
```

Comment:
Can you use Java's own `HexFormat` rather than Apache's `Hex`?  (I think we're going to eventually try to get rid of the latter's library, or so I've heard.) (`HexFormat.parseHex` also doesn't throw a checked exception, which is nice.)

Reply:
Hex was already used in the class so just used what was available without importing anything new.  Will use HexFormat in a future pr